src/server/Html.js

```diff
@@ -1,10 +1,12 @@
 /* eslint react/no-danger:0 */
 import React, {PropTypes} from 'react';
-import { LookRoot } from 'react-look';
+import {LookRoot} from 'react-look';
 import {Provider} from 'react-redux';
 import {RouterContext} from 'react-router';
 import {renderToString} from 'react-dom/server';
 
+const segmentSnippet = require('../../build/segmentSnippet.json');
```

Comment:
After sleeping on this implementation – I'm not sure I'm in love with it.

This doesn't really have any advantages over calculating the snippet once in memory (when declared in `Html.js`'s global scope.

I am going to take one more turn on this...